apps/XiaoMiTool/install

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+install_packages openjfx adb android-tools-fastboot openjdk-11-jdk || exit 1
+
+git_clone https://github.com/francescotescari/XiaoMiToolV2 -b linux || exit 1
+cd XiaoMiToolV2
+echo "Building XiaoMi Tool..."
+./gradlew build || error "Failed to build XiaoMiToolV2!"
+
+echo "Creating menu shortcut..."
+echo "[Desktop Entry]
+Name=XiaoMi Tool
+Comment=XiaoMiTool is the all-in-one tool to make modding easy for everyone with a Xiaomi smartphone.
+Exec=gksudo -k -u root /home/pi/XiaoMiToolV2/gradlew run
```

Comment:
same issue here... if the user had default java of 8 or a lower version, this will fail to run

expanding on my first suggestion to fix this as well
```
case "$arch" in
    "32") export JAVA_HOME=/usr/lib/jvm/java-11-openjdk-armhf; ./gradlew build || error "Failed to build XiaoMiToolV2!" ;;
    "64") export JAVA_HOME=/usr/lib/jvm/java-11-openjdk-arm64; ./gradlew build || error "Failed to build XiaoMiToolV2!" ;;
    *) error "Failed to detect OS CPU architecture! Something is very wrong." ;;
esac
```

and then
```
Exec=gksudo -k -u root JAVA_HOME=$JAVA_HOME /home/pi/XiaoMiToolV2/gradlew run